tox.ini

```diff
@@ -21,23 +21,14 @@ envlist = fix_lint,
           py{37,38,39,310,311,312}-{extras,unit-parallel,integ,pandas,sso},
           coverage
 skip_missing_interpreters = true
-requires =
-    tox-external-wheels>=0.1.6
 
 [testenv]
 description = run the tests with pytest under {basepython}
 extras =
     development
     pandas: pandas
     sso: secure-local-storage
-install_command = python -m pip install -U {opts} {packages}
-external_wheels =
-    py37-ci: dist/*cp37*.whl
-    py38-ci: dist/*cp38*.whl
-    py39-ci: dist/*cp39*.whl
-    py310-ci: dist/*cp310*.whl
-    py311-ci: dist/*cp311*.whl
-    py312-ci: dist/*cp312*.whl
+package = wheel
```

Comment:
Now you are introducing dependency on a tox4 feature, please update `minversion = 3.7` above in this file.